packages/web-service/src/constants.js

```diff
@@ -1,2 +1,3 @@
 export const SESSION_TTL_MS_DEFAULT = 3 * 60 * 60 * 1000
 export const SESSION_COOKIE_NAME_DEFAULT = 'sid'
+export const DEFAULT_ROLE = 'USER' // Will be replaced
```

Comment:
Is this coming in a follow up PR? Or fixed much further down the line?

Reply:
It's further down the line. We have to determine how the roles will work, we may have things like APPROVER, CREATOR, SUBMITTER, READ-ONLY etc.